pkg/apis/networking/types.go

```diff
@@ -57,8 +78,7 @@ type AppliedToGroupList struct {
 type AddressGroup struct {
```

Comment:
Do you think AddressGroup and AppliedToGroup can share the same struct?

Reply:
You mean just sharing the struct or merging them into single resource? If former, we can't as the API path comes from the struct name, they have to be two structs. If latter, it's possible, but we need to set pod references and IPs for all members?

Reply:
I mean the struct only. It is not possible to do something like "type AddressGroup PolicyGroup" either?

Reply:
i feel like that is technically possible.. i can try it out in a separate commit on this PR.. unless Quan already knows its not workable from apiserver perspective?

Reply:
I didn't think of this way and don't know whether it would work, maybe we can have a try, but do we want to keep them separated intentionally so we can add comments to explain their difference in fields, like `MemberPod` doesn't contain IP in AppliedToGroup while it doesn't contain PodReference in AddressGroup, and are they likely to be same in future too?

Reply:
Even you share the same struct, you can still comment the difference?

Reply:
i think Quan wants to keep it extensible for future differences
do you think we should do something like 
type AddressGroup struct {
    PolicyGroup
} ?

Reply:
@tnqn @jianjuns what do you guys think how we should proceed?

Reply:
IMO it would not save much code even if we declare one type equal another and we might need more comments to explain the equality of struct and difference in actual. And given that they are not really associated in struct and just happen to be same for now, should we just keep them irrelevant as it is?

Reply:
Ok. Would let you decide assuming you guys know better on K8s conventions.

Reply:
Spoke to Quan last night over this as well. Lets keep the current way to show separation between the two groups. 
@jianjuns : Do you have any other objection on this PR? I think @tnqn himself cannot give a lgtm since he also has commit in this PR

Reply:
Approved the PR.